### PR TITLE
jdownloader2: init at 2.0

### LINKED ISSUE #41321: Request for Package: 'jdownloader'
Package this for NixOS: http://jdownloader.org

---

EDIT by @cole-h (copied from dupe issue https://github.com/NixOS/nixpkgs/issues/76568 because it may help with packaging):

**Metadata**

* homepage URL: http://www.jdownloader.org/ & http://beta.jdownloader.org/
* source URL: svn://svn.jdownloader.org/jdownloader & https://svn.jdownloader.org/projects
* license: **UNKNOWN, please see discussion below**
* platforms: unix, linux, darwin, win32

### PR DESCRIPTION
**Pull request by remgodow**
###### Motivation for this change
JDownloader is a popular file downloader. Nix package downloads only bootstraping file and creates a wrapper for it. Inspired by JDownloader flatpak. Unfortunately there is no way to package it "the nix way" due to how the app is designed (mandatory selfupdate + writes configs/logs in its dir, and no way to change that). The upside is that this package should not require any updates. 
Resolves: #41321 and #76568
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (bobby285271) [inline pkgs/applications/networking/jdownloader2/default.nix:30]: `pname` + `version` is suggested.

```suggestion
  pname = "jdownloader2";
```

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (bobby285271) [inline pkgs/applications/networking/jdownloader2/default.nix:46]: Nit: One empty line is okay.

```suggestion
  };

  nativeBuildInputs = [ imagemagick ];
```

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (bobby285271) [inline pkgs/applications/networking/jdownloader2/default.nix:60]: Nit: will be nice to add more indent here.
```suggestion
    mkdir -pv $out/bin $out/share/applications
    cp ${src} $out/bin/JDownloader.jar
    cp ${wrapper} "$out/bin/jdownloader.sh"
    cp "${jddesktop}/share/applications/JDownloader 2.desktop" "$out/share/applications/JDownloader 2.desktop"

    # create icons
    for size in 16 32 48 64 72 96 128 192 512 1024; do
      mkdir -p $out/share/icons/hicolor/"$size"x"$size"/apps
      convert -resize "$size"x"$size" \
        ${icon} \
        $out/share/icons/hicolor/"$size"x"$size"/apps/jdownloader.png
    done
```

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (bobby285271) [inline pkgs/applications/networking/jdownloader2/default.nix:70]:
```suggestion
  meta = with lib; {
    homepage = "http://jdownloader.org/";
    description = "Free, open-source download management tool";
    license = licenses.gpl3;
    platforms = platforms.unix;
    maintainers = with maintainers; [ remgodow ];
  };
```

Will be nice to add yourself to `maintainers/maintainer-list.nix` in a separate commit, otherwise the CI will fail as it is not able to find the attribute `remgodow`.

BTW If you have any ideas whether this is using `gpl3Only` or `gpl3Plus`?

File: pkgs/top-level/all-packages.nix
Comment (bobby285271) [inline pkgs/top-level/all-packages.nix:25474]: Nit: one more space here will be nice.

```suggestion
  jdownloader2 = callPackage ../applications/networking/jdownloader2 { };
```

**Review by bobby285271 — COMMENTED** [review]
Thanks for your contribution! Builds and runs for me :-)

**Comment by Kranzes** [comment]
I tried to this built this, and it builds fine but jdownloader just shows a white, blank screen.

**Comment by bobby285271** [comment]
Hmmm... That's weird, I can see this interface :-/

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix]: gpl3Plus

**Comment by Kranzes** [comment]
> Hmmm... That's weird, I can see this interface :-/
> 
> 

I Figured it out, it was something with bspwm.

**Comment by remgodow** [comment]
@bobby285271 I've implemented your suggestions, it should be ready now.

**Comment by Kranzes** [comment]
The meta set is incorrect.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (bobby285271) [inline pkgs/applications/networking/jdownloader2/default.nix:65]:
```suggestion
    license = licenses.gpl3Plus;
```

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (Kranzes) [inline pkgs/applications/networking/jdownloader2/default.nix:62]:
```suggestion
  meta = with lib; {
```

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (Kranzes) [inline pkgs/applications/networking/jdownloader2/default.nix:63]:
```suggestion
    homepage = "https://jdownloader.org";
```

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/jdownloader2/default.nix:39]: Please move this into a let in.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/jdownloader2/default.nix:37]:
```suggestion
  };

  dontUnpack = true;
```

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/jdownloader2/default.nix:34]:
```suggestion
    url = "https://installer.jdownloader.org/JDownloader.jar";
```
Please upload this to web.archive.org.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/jdownloader2/default.nix:26]: Can we use makeWrapper instead?

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/jdownloader2/default.nix:50]:
```suggestion
```
Please use copyDesktopItems.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/jdownloader2/default.nix:12]: This can be inlined with copyDesktopItems.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please follow the contributing guide when doing your commits.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix]: I could not find a way, even with help from NixOS discord channel. The problem is that JDDIR value must remain unchanged/unsubstituted, so that bash can do the substitution on runtime.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix]: https://web.archive.org/web/2020*/https://installer.jdownloader.org/JDownloader.jar seems like its already there

**Comment by Kranzes** [comment]
Once you're done with writing the expression could you rebase so we only have 2 commits, $package init and maintainers

**Comment by remgodow** [comment]
OK, but wouldnt squash merge do more or less the same thing?

**Comment by Kranzes** [comment]
Might be, im not very good with git.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/jdownloader2/default.nix:26]: Please upload it to web.archive.org.

Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix]: I had to restore http://, https:// does not work for installer.jdownloader.org.
Here is the web.archive.org link: https://archive.org/details/jdownloader_202109
Do you want to use it instead of official one?

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (SuperSandro2000) [reply pkgs/applications/networking/jdownloader2/default.nix:27]: Then please update the URL with that link.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix]: Url replaced with archive.org one

**Comment by milahu** [comment]
nitpick: rename to `jdownloader`?

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (milahu) [inline pkgs/applications/networking/jdownloader2/default.nix:40]: imho, "FOSS" is not relevant here. i would just say "Download manager"
(you can download all kinds of files with jd, also youtube videos ...)

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (milahu) [inline pkgs/applications/networking/jdownloader2/default.nix:25]: we dont need https, we have sha256 for integrity

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (milahu) [inline pkgs/applications/networking/jdownloader2/default.nix:13]: personally, i am NOT happy with auto-updates. should be "default off, enable on demand"

Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix:13]: So to begin with, JDownloader supports many websites via its plugin system, and those are always autoupdated. Main app does not update automatically, only checks for updates, here is more info
https://support.jdownloader.org/Knowledgebase/Article/View/updates-update-behavior

I did not try to build from source, and I'm not sure if it is possible without access to the main repo, for example they use proprietary libraries, and I believe there are closed source parts as well (kinda like VSCode). That would require some tinkering, as for ex. AUR packages it using the bootstrapper as well.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix:40]: I took that from the official website, the FOSS refers to JDownloader not the stuff you can download, I think. Sure thing, I can change that.

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix:13]: Here is an issue from the mirror about missing dependencies from password-protected official repo. https://github.com/mirror/jdownloader/issues/47

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (remgodow) [reply pkgs/applications/networking/jdownloader2/default.nix:40]: As for the package name, I can change that as well. For quite some time there were jdownloader and jdownloader2, the jdownloader is obsolete by now, but few people still use the jdownloader2 name as you can see on the repology https://repology.org/projects/?search=jdownloader

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (milahu) [reply pkgs/applications/networking/jdownloader2/default.nix:13]: > Here is an issue from the mirror about missing dependencies

the svn repos listed in https://jdownloader.org/knowledge/wiki/development/get-started are public, the issue is only a request to mirror these repos

> they use proprietary libraries, and I believe there are closed source parts as well

im afraid yes ... im trying to build from source with [headless eclipse](https://gnu-mcu-eclipse.github.io/advanced/headless-builds/), but at the end of the day, there will still be some binary blobs (`*.jar` files with closed source), so jdownloader has license `unfree`

&rarr; use containers for better security?

[pyload](https://github.com/pyload/pyload) is missing : /

```
$ nix-shell -p pyload 
error: pyload has been removed from nixpkgs, as it was unmaintained.
```

Comment (milahu) [reply pkgs/applications/networking/jdownloader2/default.nix:13]: > → use containers for better security?

https://github.com/mviereck/x11docker

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (milahu) [inline pkgs/applications/networking/jdownloader2/default.nix:52]: source icon has only size 512

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (milahu) [reply pkgs/applications/networking/jdownloader2/default.nix:13]: > x11docker

https://github.com/jlesage/docker-jdownloader-2 has invented this wheel already : )

**Comment by vs49688** [comment]
Just thought I'd mention I have one here that I'm using:
https://github.com/vs49688/nix-repo/blob/master/pkgs/jdownloader/default.nix

* With custom patches to make it use the XDG_* dirs instead of trying to write to its local directory.
* I've also been in contact with Daniel (jiaz) from JDownloader in order to get something proper implemented in regards to this (JD Ticket #ADF-935-28292). He explicitly asked to hold off on packaging until he can get proper handling in place, but can't give an ETA:

```
Thanks again for detailed feedback :)
I will setup a VM and try to get familiar with it, so I can test my changes :)
But I can't promise when I will find time, how fast I will be.

I will ping you when there is progress/are questions, okay?

daniel


Ticket Details
Ticket ID: ADF-935-28292
Department: Development
Type: Issue
Status: Todo
Priority: Low

Support Center: https://support.jdownloader.org
```

File: pkgs/applications/networking/jdownloader2/default.nix
Comment (milahu) [reply pkgs/applications/networking/jdownloader2/default.nix:13]: > at the end of the day, there will still be some binary blobs (`*.jar` files with closed source), so jdownloader has license `unfree`

yepp

```
Compiling 4610 source files to /build/jdownloader/bin
/build/jdownloader/src/jd/plugins/AccountInfo.java:267: error: cannot find symbol
        final long serverTime = br.getCurrentServerTime(-1);
                                  ^
  symbol:   method getCurrentServerTime(int)
  location: variable br of type Browser
```

jdownloader requires closed sources:

```
JDClosed
JDownloaderServerOptions
UpdateClientV2
JDownloaderUpdater
```

per [jdownloader/build/newBuild/build.xml](https://github.com/mirror/jdownloader/blob/master/build/newBuild/build.xml)

i want to patch the myjdownloaderclient ...

in headless mode, jdownloader requires a valid account for myjdownloader.
i have not-yet found a way to use a self-hosted api server, or to disable this completely ...
so much for "untrusted" software

relevant configkey is [serverhost](https://github.com/mirror/jdownloader/blob/1319dcb620976270cab3c32aedf54e2c3b0884dd/src/org/jdownloader/api/myjdownloader/MyJDownloaderSettings.java#L34) (default `api.jdownloader.org`) in `cfg/org.jdownloader.api.myjdownloader.MyJDownloaderSettings.json`
but jd always says

```
|---------------------------Headless Information-------------------------------
|       MyJDownloader
|       Your 'My JDownloader' logins are not correct. 
|       Please check username/email and password!
|       Enter y -> Enter Logins
|       Enter n -> Exit JDownloader
```

so im back to binary patching ... or `pyLoad`

**Comment by AndersonTorres** [comment]
Drafting this until news from JDownloader upstream.

**Comment by AndersonTorres** [comment]
Drafted until news.

**Comment by AndersonTorres** [comment]
Closed

**Comment by vs49688** [comment]
Quick update. I've been in contact with upstream, and they're adding a build-time option to enable using XDG_* and friends, and are going to provide an endpoint to query the stable SVN revisions for updating.

Will report back once I hear more.

**Comment by milahu** [comment]
using `~/.local/share/jdownloader` instead of `~/.jd` is nitpickery

jdownloader practically needs autoupdate, as it breaks often

the elephant in the room is `JDClosed` in [build.xml](https://github.com/mirror/jdownloader/blob/master/build/newBuild/build.xml)

```xml
	<property name="dep.jdc" value="../JDClosed" />
```

which is a known wontfix

> "jdownloader requires closed sources"
> -> Yes, it is true that our official build also contains closed source parts
> 
> Repository: JDClosed
> -> plugin sources that are not public due to the sites looking for plugin changes to break them again and again and again.
> of course we can provide an up2date zip of the sources or even provide access to it but we don't want those parts to be public as
> this just will cause the plugins to break (very) fast and end up in cat/mice game.

&rarr; im running jd in docker https://github.com/NixOS/nixpkgs/issues/41321#issuecomment-1120217437

**Comment by milahu** [comment]
```
$ find trunk/ browser/ MyJDownloaderClient/ -name '*.jar' | wc -l 
76
```

still lots of room to hide malicious binaries. this is not "open source"

**Comment by vs49688** [comment]
Baby steps. Upstream's been receptive to changes so far, I don't want to demand too much of them at once.

**Comment by milahu** [comment]
> for "casual" users like myself, who don't download from "hostile" sites very often, the open-source version is fine.

did you try [pyload](https://github.com/milahu/nur-packages/tree/master/pkgs/pyload)?

**Comment by vs49688** [comment]
> other binaries:
> 
> ```
> $ git ls-tree -r -t -l --full-name HEAD | sort -n -k 4 -r >large-files.txt
> $ head -n 100 large-files.txt | cut -d$'\t' -f2 | grep -v -e 'zip$' -e 'png$' -e '\.js$' -e 'psd$' | head 
> trunk/ressourcen/tools/linux/ffmpeg/x64/ffmpeg
> trunk/ressourcen/tools/linux/ffmpeg/x64/ffprobe
> trunk/ressourcen/tools/mac/ffmpeg_10.10+/libavcodec.58.dylib
> trunk/ressourcen/tools/linux/ffmpeg/i386/ffmpeg
> trunk/ressourcen/tools/linux/ffmpeg/i386/ffprobe
> trunk/ressourcen/tools/Windows/ffmpeg/x64/avcodec-58.dll
> trunk/ressourcen/tools/Windows/ffmpeg/i386/avcodec-58.dll
> trunk/ressourcen/tools/mac/ffmpeg_10.5.x-/ffmpeg
> trunk/ressourcen/tools/mac/ffmpeg_10.5.x-/ffprobe
> trunk/ressourcen/tools/mac/ffmpeg_10.6+/ffmpeg
> ```
> 

Yup, they're going to make it search $PATH for ffmpeg and friends, so these can just be rm'd for our purposes.


> > for "casual" users like myself, who don't download from "hostile" sites very often, the open-source version is fine.
> 
> did you try [pyload](https://github.com/milahu/nur-packages/tree/master/pkgs/pyload)?

I've never heard of it, I'll have a look.

**Comment by airone01** [comment]
For posterity, the JDownloader repo is accessible _without a password_ using `subversion` at `svn://svn.jdownloader.org/jdownloader`.
The file [`MyJDownloaderClient/License My JDownloader Client.txt`](https://github.com/user-attachments/files/27417649/License.My.JDownloader.Client.txt) mentions AGPL 3.0 for noncommercial usage.
This same file also says
```
=== 3rd Party Licences ===
    Some parts of the [The Product] use or reference 3rd party libraries and classes. These parts may have different licensing conditions. Please check the *.license and *.info files of included libraries
    to ensure that they are compatible to your use-case. Further more, some *.java have their own license. In this case, they have their license terms in the java file header.
```

<details><summary><code>find -type f -name '*.info' -o -name '*.license'</code></summary>
<p>

```
find -type f -name '*.info' -o -name '*.license'
./trunk/themes/themes/simple/org/jdownloader/images/icons8.license
./trunk/themes/themes/standard/org/jdownloader/images/black-friday.png.license
./trunk/themes/themes/standard/org/jdownloader/images/beer.png.license
./trunk/themes/themes/standard/org/jdownloader/images/modem.png.license
./trunk/themes/themes/standard/org/jdownloader/images/valentine.png.license
./trunk/themes/themes/standard/org/jdownloader/images/rar.png.license
./trunk/themes/themes/standard/org/jdownloader/images/mediaplayer.png.license
./trunk/themes/themes/standard/org/jdownloader/images/xmas-gift.png.license
./trunk/themes/themes/standard/org/jdownloader/images/log.png.license
./trunk/themes/themes/standard/org/jdownloader/images/halloween.png.license
./trunk/themes/themes/standard/org/jdownloader/images/file.png.license
./trunk/themes/themes/standard/org/jdownloader/images/test.png.license
./trunk/themes/themes/standard/org/jdownloader/images/extract.png.license
./trunk/themes/themes/standard/org/jdownloader/images/champagne.png.license
./trunk/themes/themes/standard/org/jdownloader/images/thumbs-up.png.license
./trunk/themes/themes/standard/org/jdownloader/images/easter-egg.png.license
./trunk/themes/themes/standard/org/jdownloader/images/harddisk.png.license
./trunk/themes/themes/standard/org/jdownloader/images/console.png.license
./trunk/themes/themes/standard/org/jdownloader/images/desktop.png.license
./trunk/themes/themes/standard/org/jdownloader/images/thumbs-down.png.license
./trunk/themes/themes/standard/org/jdownloader/images/exttable/widthLocked.png.license
./trunk/themes/themes/standard/org/jdownloader/images/exttable/resetColumns.png.license
./trunk/themes/themes/standard/org/jdownloader/images/exttable/columnButton.png.license
./trunk/themes/themes/standard/org/jdownloader/images/confirmSelectedLinks.png.license
./trunk/themes/themes/standard/org/jdownloader/images/restart.png.license
./trunk/themes/themes/standard/org/jdownloader/images/logo/ffmpeg.png.license
./trunk/themes/themes/standard/org/jdownloader/images/logo/dlc.png.license
./trunk/themes/themes/standard/org/jdownloader/images/logo/nzb.svg.license
./trunk/themes/themes/standard/org/jdownloader/images/speed.png.license
./trunk/themes/themes/flat/org/jdownloader/images/icons8.license
./trunk/ressourcen/libs/sqljet.jar.info
./trunk/ressourcen/libs/jna.jar.info
./trunk/ressourcen/libs/svnkit.jar.info
./trunk/ressourcen/libs/UPNP/cling-support.jar.info
./trunk/ressourcen/libs/UPNP/seamless-xml.jar.info
./trunk/ressourcen/libs/UPNP/cling-core.jar.info
./trunk/ressourcen/libs/UPNP/seamless-util.jar.info
./trunk/ressourcen/libs/UPNP/seamless-http.jar.info
./trunk/ressourcen/libs/dbus/libdbus-java.jar.info
./trunk/ressourcen/libs/irclib.jar.info
./trunk/ressourcen/libs/laf/synthetica.license
./trunk/ressourcen/libs/laf/flatlaf.license
./trunk/ressourcen/libs/zip4j.jar.info
./trunk/ressourcen/libs/js.jar.info
./trunk/ressourcen/libs/jna_platform.jar.info
./trunk/ressourcen/libs/jsyntaxpane.jar.info
./trunk/ressourcen/libs/sevenzipjbindings.jar.info
./trunk/ressourcen/licenses/image4j.license
./trunk/ressourcen/licenses/apache2.license
./trunk/ressourcen/licenses/cobra.license
./trunk/ressourcen/licenses/icons8.license
./trunk/ressourcen/licenses/jdownloader.license
./trunk/ressourcen/licenses/rhino_js.license
./trunk/ressourcen/licenses/synthetica.license
./trunk/ressourcen/licenses/Base64.license
./trunk/ressourcen/licenses/MIT.license
./trunk/ressourcen/licenses/irclib.license
./trunk/ressourcen/licenses/miglayout.license
./trunk/ressourcen/licenses/XUI.license
./trunk/ressourcen/licenses/cling.license
./trunk/ressourcen/licenses/batik.license
./trunk/ressourcen/licenses/hqsql.license
./trunk/ressourcen/licenses/clink.license
./trunk/ressourcen/licenses/Jetbrains - intellij.license
./trunk/ressourcen/licenses/apache-sanselan.license
./trunk/ressourcen/licenses/proxyVole.license
./trunk/ressourcen/licenses/filters.license
./trunk/ressourcen/licenses/htmlcleaner.license
./trunk/ressourcen/tools/Windows/ffmpeg/i386/ffmpeg.info
./trunk/ressourcen/tools/Windows/ffmpeg/x64/ffmpeg.info
./trunk/ressourcen/tools/mac/ffmpeg_10.5.x-/ffprobe.info
./trunk/ressourcen/tools/mac/ffmpeg_10.5.x-/ffmpeg.info
./trunk/ressourcen/tools/mac/ffmpeg_10.10+/ffmpeg.info
./trunk/ressourcen/tools/mac/ffmpeg_10.6+/ffprobe.info
./trunk/ressourcen/tools/mac/ffmpeg_10.6+/ffmpeg.info
./trunk/ressourcen/tools/linux/ffmpeg/i386/ffmpeg.info
./trunk/ressourcen/tools/linux/ffmpeg/x64/ffmpeg.info
```

</p>
</details>

**Pull request closed without merging**
